spec/services/find/cycle_timetable_spec.rb

```diff
@@ -50,6 +50,22 @@ module Find
       end
     end
 
+    describe '.find_opens(year)' do
+      it 'returns find_opens date for 2021' do
+        Timecop.travel(Time.zone.local(2021, 1, 1, 12, 0, 0)) do
+          expect(described_class.find_opens).to eq(Time.zone.local(2020, 10, 6, 9))
+        end
+      end
+
+      context 'passing 2024 as argument' do
+        it 'returns find_opens date for 2024' do
+          Timecop.travel(Time.zone.local(2021, 11, 1, 12, 0, 0)) do
```

Comment:
Unless I have misunderstood, I find it confusing that we are travelling to 2021 here and then expecting find to open in 2023

Reply:
It's testing that passing 2024 to find_opens when the year is 2021 that the correct find_opens date is returned for the 2024 recruitment cycle. I'm happy to change the years or whatever if you like

Reply:
That kind of makes sense. I think it's fine. It may potentially cause even more confusion if the date makes logical sense. 

I know that sounds weird.